tests/test_ext_commands.py

```diff
@@ -477,6 +478,24 @@ def test_raises_invalid_repo_if_invalid_repo(
             exc_info.value
         )
 
+    def test_commit_message(self, sanitizer_config, fake_repo):
+        target_branch = "student-version"
+        commit_message = "This is not the default!"
+
+        # Just split() ruins our commit_message, so we split on
+        # hashtag instead of whitespace
+        run_repobee(
+            shlex.split(
+                f"sanitize repo --target-branch {target_branch} "
+                f"--repo-root {fake_repo.path} "
+                f"--commit-message '{commit_message}'"
+            )
+        )
+
+        fake_repo.repo.git.checkout(target_branch)
+        commit = fake_repo.repo.head.commit
+        assert commit.message == (commit_message + "\n")
```

Comment:
If we don't care about trailing and leading whitespace, I'd do this instead:

```suggestion
        assert commit.message.strip() == commit_message.strip()
```